admin_guide/tcp_ingress_external_ports.adoc

```diff
@@ -0,0 +1,84 @@
+[[admin-guide-expose-external-ports]]
+= Enabling Ingress Traffic by Exposing External Ports
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+In {product-title} 3.2, you could use `*externalIP*` in the router service to control which external IP addresses expose the port. However, the OpenShift router could only listen on ports 80 and 443 for routing and mapping to backends (which have protocols). Previously, routes did not enable you to specify an external port. Port 80 runs in HTTP mode and 443 runs in TCP mode.
+
+However, you may want to expose external TCP traffic to the outside world.
+Starting in {product-title} 3.3, cluser administrators can expose an external
+port, enabling ingress traffic (TCP/UDP) routing to a definable backend.
+
+Exposing external ports allows you to allocate a unique IP address per service,
+rather than trying to manage the port space of a limited number of shared IP
+addresses. This ensures that each service can simply direct traffic from its
+chosen ports, regardless of the ports selected by other services.
+
+There are benefits to using `*IngressIP*` instead of `*externalIP*`:
+
+- An ingress IP is intended to be automatically assigned, and can only be assigned to one service. 
+- An external IP must be manually assigned, and conflicting assignments cannot be detected. Unfortunately, multiple services could potentially request the same IP, and the conflict would go undetected.
+
+== Restrictions
+
+To use `*IngressIP*`, you can either:
+
+- Select an IP address with the existing `*ExternalIP*` ingress.
```

Comment:
I got that from this comment on the Trello card: 

**Ben Bennett**
_@jorgemoralespou: Maru posted openshift/origin: Pull Request 9454 and based on the feedback_
_is rewriting it. At the moment the strategy is that you can either choose an IP address with_
_the existing ExternalIP ingress, or if you want an IP address to be assigned from a pool, then_ 
_the new feature will implement a bare-metal version of the LoadBalancer service type and assign_ _IP addresses to the services. You will need to ensure that the IP address pool you assign_ _terminates at a node on your cluster. You can use the existing 'oadm ipfailover' to configure ip_ _failover for the external IPs._

Apologies if it's wrong to include that here, but the Trello card had zero information on what needed to be documented so I just did my best with what was there to use as a starting point.